depends/packages/boost.mk

```diff
@@ -1,10 +1,29 @@
 package=boost
-$(package)_version=1.81.0
-$(package)_download_path=https://archives.boost.io/release/$($(package)_version)/source/
-$(package)_file_name=boost_$(subst .,_,$($(package)_version)).tar.gz
-$(package)_sha256_hash=205666dea9f6a7cfed87c7a6dfbeb52a2c1b9de55712c9c1a87735d7181452b6
+$(package)_version = 1.88.0
+$(package)_download_path = https://github.com/boostorg/boost/releases/download/boost-$($(package)_version)
+$(package)_file_name = boost-$($(package)_version)-cmake.tar.gz
+$(package)_sha256_hash = dcea50f40ba1ecfc448fdf886c0165cf3e525fef2c9e3e080b9804e8117b9694
+$(package)_patches = skip_compiled_targets.patch
+$(package)_build_subdir = build
+
+define $(package)_set_vars
+  $(package)_config_opts = -DBOOST_INCLUDE_LIBRARIES="multi_index;signals2;test"
```

Comment:
Did I forget to explicitly mention the [`tuple`](https://www.boost.org/doc/libs/latest/libs/tuple/doc/html/tuple_users_guide.html) library?

See: https://github.com/bitcoin/bitcoin/blob/f999c3775c12ef732a1920dd52257a8cd34cdcc8/src/txrequest.cpp#L18

Reply:
That include should be perfectly safe to remove, btw. It's included by (at least) `boost/multi_index/sequenced_index.hpp`.